packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -435,13 +438,18 @@ export class ProjectActions {
     const [newSpec] = codeGenResults.files
 
     const spec = this.ctx.file.normalizeFileToSpec({
-      absolute: newSpec.file,
+      absolute: path.join(project.projectRoot, newSpec.file),
```

Comment:
The `newSpec.file` should be an absolute path. Looks like the `packages/app/src/specs/generators/component/ComponentGeneratorStepOne.vue` component is passing in a relative path to the mutation when it should be an absolute, which is why you had to add the line above.

Reply:
Yep, you are right. Fixed.